addons/l10n_din5008_sale/report/din5008_sale_templates.xml

```diff
@@ -8,38 +8,38 @@
                         <t t-if="doc.state in {'draft', 'sent'}">
                             <tr t-if="doc.name">
                                 <td>Quotation No.:</td>
-                                <td><t t-out="doc.name"/></td>
+                                <td><span class="m-0" t-field="doc.name"/></td>
```

Comment:
... like so : 

```suggestion
                                <td><div t-field="doc.name"/></td>
```

Could you try to see if the results are the same ?

Reply:
yes you're right 
i updated it